collection_manager/collection_manager/main.py

```diff
@@ -8,7 +8,7 @@
 
 logging.basicConfig(level=logging.INFO)
 logging.getLogger("pika").setLevel(logging.WARNING)
-logger = logging.getLogger("collection_manager")
+logger = logging.getLogger()
```

Comment:
I am usually doing:
logger = logging.getLogger(__name__)

Does it behave the same without the __name__ argument ?

Reply:
good catch -- fixed.